test/core/ref_func.wast

```diff
@@ -93,6 +93,7 @@
   (elem (table $t) (i32.const 0) funcref (ref.func $f4))
   (elem func $f5)
   (elem funcref (ref.func $f6))
+  (elem declare func $f1 $f2 $f3 $f4 $f5 $f6)
```

Comment:
The point of this test is that this is no longer needed, per the resolution of #76. The occurrences in the other declarations are sufficient.

Reply:
Ah, I missed that. Thanks! I removed the change.